app/jsx/components/TabsComp.jsx

```diff
@@ -19,7 +19,7 @@ class TabsComp extends React.Component {
   componentWillReceiveProps(nextProps) {
     if (this.props.currentTab != nextProps.currentTab) { 
       // For keyboard users, jump to heading within the tab.
-      setTimeout(()=>$(".c-tabcontent__main-heading").focus(), 0)
+      setTimeout(() =>$(".c-tabcontent__main-heading").focus(), 0)
```

Comment:
Colors don't format well in my version of Sublime Text unless these have spaces before the '='

Reply:
Interesting. Good to know!